EMacco/Customer-Ticketing-System
# #168728299 Create request

#### What does this PR do?
This PR allows a user to be able to log(create) a request

#### Description of Task to be completed?
- Create a route `POST /requests`
- Validate the request body
- Only an authenticated user can create a request
- Return details of the request
- Write tests

#### How should this be manually tested?
- Checkout to this branch
- Open Postman and make a `POST /requests` request containing `title` and `description`
- You should see that the request was successfully created 

#### What are the relevant pivotal tracker stories?
[#168728299](https://www.pivotaltracker.com/story/show/168728299)
